### PR TITLE
[eclipse/xtext#1171] Move deprecated classes to separate source folder

### PR DESCRIPTION
**Pull request by kthoms**
Signed-off-by: Karsten Thoms <karsten.thoms@itemis.de>

### CODE REVIEW

File: org.eclipse.xtext.xbase.lib.tests/build.gradle [hunk 1]
```diff
@@ -12,3 +12,5 @@ jar {
 		include 'about_org.eclipse.xtext.xbase.lib.html'
 	}
 }
+
```
Comment (cdietrich) [inline org.eclipse.xtext.xbase.lib.tests/build.gradle]: build.properties

File: org.eclipse.xtext.xbase.lib/build.gradle [hunk 2]
```diff
@@ -6,4 +6,6 @@ dependencies {
 	compile "com.google.guava:guava:$versions.guava"
 }
 
-apply from: "${rootDir}/gradle/gwtSourceProject.gradle"
\ No newline at end of file
+apply from: "${rootDir}/gradle/gwtSourceProject.gradle"
+
```
Comment (cdietrich) [inline org.eclipse.xtext.xbase.lib/build.gradle]: sorry herer buiuld.properties

Comment (kthoms) [reply org.eclipse.xtext.xbase.lib/build.gradle]: This project has no build.properties. What to do?

Comment (cdietrich) [reply org.eclipse.xtext.xbase.lib/build.gradle]:
```
[dietrich@dietrich-thinkpad-x230 xtext-lib]$ find . -name "build.properties"
./releng/org.eclipse.xtend.gwt/build.properties
./releng/org.eclipse.xtext.xbase.lib/build.properties
./releng/org.eclipse.xtext.license/build.properties
./releng/org.eclipse.xtext.gwt/build.properties
```  
i did miss these are the features

**Pull request merged**